website/docs/cli/code/index.mdx

```diff
@@ -17,33 +15,32 @@ in the Terraform language.
 ## Workflows 
 
 Terraform CLI includes several commands to make Terraform code more convenient
-to work with. Integrating these commands into your editing workflow can
-potentially save you time and effort.
+to work with. The following commands can be integrated into your editing 
+workflow and may save you time and effort:
```

Comment:
I know this was in the original, but I'd suggest we make this a little less vague and more direct:
```suggestion
workflow to test expressions and format and validate your configuration syntax:
```